hail_search/hail_search_query.py

```diff
@@ -683,7 +689,34 @@ def _sort_order(self, ht):
         return sort_expressions
 
     def _get_sort_expressions(self, ht, sort):
-        return self.SORTS[sort](ht)
+        sort_expressions = []
+        if sort in self.SORTS:
+            sort_expressions = self.SORTS[sort](ht)
+
+        elif sort in POPULATION_SORTS:
```

Comment:
Should `POPULATION_SORTS` be present as a class var in some capacity?  Maybe by combining with `self.POPULATIONS`?